codefresh.yml

```diff
@@ -24,7 +24,7 @@ steps:
       - "apt-get update"
       - "apt-get install -y wget git curl"
       - 'curl -s https://api.github.com/repos/nobl9/sloctl/releases/latest | grep "browser_download_url.*linux*" | cut -d : -f 2,3 | tr -d \" |  wget -qi -'
-      - "cf_export VERSION=$(git describe --tags --abbrev=0)"
+      - 'cf_export VERSION=$(curl -s https://api.github.com/repos/nobl9/sloctl/releases/latest | grep "tag_name" | cut -d : -f 2,3 | tr -d \" | tr -d ,)'
```

Comment:
I would actually use `jq` tool to extract data from this, as it is json, and grep/cut/etc may not always get you the right results - some unexpected character may break this.
That should also be used for the line above.

Reply:
Thanks for the feedback. This has been updated!